notebooks/eval_notebook.ipynb

```diff
@@ -0,0 +1,721 @@
+      "Processing model llama2-800k\n",
+      "Processing model llama2-1.6m\n",
+      "Processing model llama2-3.2m\n",
+      "Processing model llama2-6.4m\n",
+      "Processing model llama2-12.8m\n",
+      "Processing model llama2-25.6m\n"
+     ]
+    }
+   ],
+   "source": [
+    "from delphi.eval.calc_model_group_stats import calc_model_group_stats\n",
+    "\n",
+    "\n",
+    "token_groups = {id:{'selected':id in selected_ids} for id in vocab.values()}\n",
+    "model_group_stats = calc_model_group_stats(\n",
+    "    list(tokenized_corpus_dataset), next_logprobs_plot, token_groups, ['selected']\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 22,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "b72e05f2a72d481bba57e94fd9e5540d",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/plain": [
+       "VBox(children=(Dropdown(description='Token Category:', options=('selected',), value='selected'), FigureWidget(…"
+      ]
+     },
+     "execution_count": 22,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "from collections import defaultdict\n",
+    "from delphi.eval.vis_per_token_model import visualize_per_token_category\n",
+    "\n",
+    "performance_data = defaultdict(dict)\n",
+    "for model in models:\n",
+    "    stats = model_group_stats[(model, 'selected')]\n",
+    "    performance_data[model]['selected'] = (\n",
+    "        -stats[\"median\"],\n",
+    "        -stats[\"75th\"],\n",
+    "        -stats[\"25th\"],\n",
+    "    )\n",
+    "\n",
+    "# TODO: this is using the older version of the plotting func\n",
+    "visualize_per_token_category(\n",
+    "    performance_data,\n",
+    "    log_scale=True\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 44,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "2c2887ef72ec495ebe69552edef2ea47",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/plain": [
+       "interactive(children=(FloatRangeSlider(value=(0.25, 0.75), description='Start quantile', max=1.0, step=0.01), …"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "data": {
+      "text/plain": [
+       "<function __main__.show_pos_map(quantile: tuple[float, float], model_name_1: str, model_name_2: str, label: str, samples: int)>"
+      ]
+     },
+     "execution_count": 44,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "from ipywidgets import interact\n",
+    "from delphi.eval.token_positions import get_all_tok_metrics_in_label\n",
+    "from delphi.eval.vis import vis_pos_map\n",
+    "import ipywidgets as widgets\n",
+    "\n",
+    "\n",
+    "\n",
+    "def show_pos_map(\n",
```

Comment:
Similar to the above visualization function, I wonder if it is possible to "hook" the token_ids selector UI here for interactivity.